core/src/main/kotlin/io/saagie/updatarium/dsl/ChansetSet.kt

```diff
@@ -53,11 +53,13 @@ data class ChangeSet(
                     it.execute()
                 }
                 InMemoryAppenderManager.stopRecord()
-                persistEngine.unlock(this, Status.OK,InMemoryAppenderAccess.getEvents())
+                persistEngine.unlock(this, Status.OK, InMemoryAppenderAccess
+                    .getEvents(persistConfig = persistEngine.configuration,success = true))
```

Comment:
missing space between persistEngine.configuration and success = false

Reply:
Fixed